### PR TITLE
influxctl 2.9.0

### PR DESCRIPTION
**Pull request by sanderson**
Includes release notes and all updated changes for `influxctl` 2.9.0

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/query.md [hunk 1]
```diff
@@ -52,10 +52,10 @@ Default: `table`.
 When using the `table` format, by default, timestamps are formatted as RFC3339
 timestamps. Use the `--time-format` flag to specify one of the available time formats:
 
-- `rfc3339`: _(Default)_
+- `rfc3339nano`: _(Default)_
   [RFC3339-formatted timestamp](/influxdb/cloud-dedicated/reference/glossary/#rfc3339-timestamp)--for example:
   `2024-01-01T00:00:00.000000000Z`
-- `unix`: [Unix nanosecond timestamp](/influxdb/cloud-dedicated/reference/glossary/#unix-timestamp)
+- `unixnan`: [Unix nanosecond timestamp](/influxdb/cloud-dedicated/reference/glossary/#unix-timestamp)
```
Comment (powersj) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/query.md:58]: unixnano

Comment (powersj) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/query.md:58]: Did you also want to update the reference to point to rfc3339-nano as well?

Same question for clustered.

Comment (sanderson) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/query.md]: Corrected and added.

**Review by powersj — APPROVED** [review]
Thank you again!

**Pull request merged**